src/stylesheets/base/_typography.scss

```diff
@@ -0,0 +1,19 @@
+@font-face {
+    font-family: geomanist;
+    src: url(fonts/geomanist-regular-webfont.woff2);
+}
+
+@font-face {
+    font-family: geomanist-medium;
+    src: url(fonts/geomanist-medium-webfont.woff2);
+}
+
+body {
+  font-family: 'geomanist';
+  font-size: 16px;
+  color: #504949;
```

Comment:
it's good practice to keep all your color variables in one file together to see if you are using lots of similar colors. sometimes designers may pick up a project from another designer and mock out a button or something that isn't the same color as what has been established on the site. putting all colors as variables into a single file allows you to scan all your colors before adding a new one. it's good to add it even if you think you are going to use the color only once.